.npm-upgrade.json

```diff
@@ -46,27 +46,27 @@
     },
     "react": {
       "versions": "",
-      "reason": "MM-34187 - React updates are handled separately from regular dependencies."
+      "reason": "MM-45255 - React updates are handled separately from regular dependencies."
     },
     "@types/react": {
       "versions": "",
-      "reason": "MM-34187 - React updates are handled separately from regular dependencies."
+      "reason": "MM-45255 - React updates are handled separately from regular dependencies."
     },
     "react-dom": {
       "versions": "",
-      "reason": "MM-34187 - React updates are handled separately from regular dependencies."
+      "reason": "MM-45255 - React updates are handled separately from regular dependencies."
     },
     "@types/react-dom": {
       "versions": "",
-      "reason": "MM-34187 - React updates are handled separately from regular dependencies."
+      "reason": "MM-45255 - React updates are handled separately from regular dependencies."
```

Comment:
0/5 on removing these fields from here altogether, since we batch updates in separate PRs anyways

Reply:
Since React 18 is out and we aren't going to want to update these during the regular monthly updates, I think it makes sense to keep these here